velox/docs/functions/array.rst

```diff
@@ -28,6 +28,13 @@ Array Functions
         SELECT array_distinct(ARRAY [1, 2, 1]); -- [1, 2]
         SELECT array_distinct(ARRAY [1, NULL, NULL]); -- [1, NULL]
 
+.. function:: array_duplicates(array(E)) -> array(E)
+
+    Returns a set of elements that occur more than once in array.
+    E must be coercible to bigint or varchar.
```

Comment:
I'd drop "coercible to" since Velox doesn't perform implicit coercions.

Reply:
I expect Presto's query planner to do that. You can test it out by adding an e2e test.